test/e2e/kubectl/kubectl.go

```diff
@@ -801,6 +803,67 @@ metadata:
 			framework.ExpectNoError(c.CoreV1().Pods(ns).Delete(ctx, "run-test-3", metav1.DeleteOptions{}))
 		})
 
+		ginkgo.It("should support inline execution and attach with websockets", func(ctx context.Context) {
```

Comment:
"with websockets ... or fallback to spdy"

Reply:
Fixed.